src/System.Private.ServiceModel/tools/scripts/StartWCFSelfHostedSvcDoWork.cmd

```diff
@@ -4,9 +4,12 @@ setlocal
 REM This script should always be called from StartWCFSelfHostedSvc.cmd
 
 set _setuplog=%~dp0..\..\..\..\SelfHostedWcfServiceSetup.log
+set _setupMonitorFile=%~dp0..\..\..\..\SelfHostedWcfServiceMonitor.log
```

Comment:
Reviews: I'm open to other ideas, but I used a file as a monitor to solve the following problem:  our CMD to start the SelfHost service launches the EXE *and blocks*.  After the EXE exits, the CMD resumes and cleans up.  This means that when hooking up the clean to kill the SelfHost exe, it also allows the startup CMD to resume and finish the cleanup.  This means log files and the bin folder are still in use *after* the SelfHost .exe has been killed.

In this PR, I introduced a short-lived monitor file that is created when the service is started by this script, and deleted when this script has finished cleanup.  That allows the new script to block the clean until this monitor is cleared.

I believe the right fix is probably making an endpoint on the util service to cleanup and shutdown so that we don't have these chained back-to-back CMD files exchanging control.  It makes it much simpler to ask for a clean shutdown in exactly one way.